pywhat/helper.py

```diff
@@ -29,7 +30,15 @@ def load_regexes() -> list:
     path = "Data/regex.json"
     fullpath = os.path.join(os.path.dirname(os.path.abspath(__file__)), path)
     with open(fullpath, "r", encoding="utf-8") as myfile:
-        return json.load(myfile)
+        regexes = json.load(myfile)
+    for regex in regexes:
+        regex["Boundaryless Regex"] = re.sub(
+            r"(?<!\\)\^(?![^\[\]]*(?<!\\)\])", "", regex["Regex"]
```

Comment:
Now that regexes are in `^(...)$` format you could again use `removepreffix` and `removesuffix` right? If `(?i)` is causing issues it can be moved after `^`.

Reply:
Yes, I can, but it means that we are forcing everyone to use this new format. Removing boundaries with regex seems to be a clever way that does not have any assumptions about the regex format.

Reply:
Hm true. We could make a test that would check the regex format which would make it easier to ensure that regexes in the database and from PRs are correct.

Reply:
I would also like to see tests that make sure the regex data is sorted by rarity 👀  Homebrew has similar tests but this might be overkill haha

Reply:
Actually a great idea

Reply:
After this gets merged, I can work on tests 😄